src/ixdat/techniques/ms.py

```diff
@@ -335,15 +335,97 @@ def __init__(
         """
         self.verbose = verbose
         self.l_cap = l_cap
+        self.l_cap_eff = {}
```

Comment:
cool idea! I think some people at a certain mass spec inlet company would be excited to learn whether `l_cap_eff` is different for different molecules.